varsim.py

```diff
@@ -271,10 +269,25 @@ def run_randdgv(dgv_file, out_vcf_fd, log_file_fd, seed, sex, options, reference
 
     return p_rand_dgv
 
-
-def get_version():
-    return subprocess.check_output("java -jar {} -version".format(VARSIMJAR), shell=True).strip()
-
+def run_bgzip(vcf):
```

Comment:
Since you are sorting and compressing, rename it to `sort_and_compress` or something like that.